views/templates/front/payment_infos.tpl

```diff
@@ -0,0 +1,32 @@
+{*
+ * 2011-2021 Blockonomics
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@blockonomics.co so we can send you a copy immediately.
+ *
+ * @author    Blockonomics Admin <admin@blockonomics.co>
+ * @copyright 2011-2021 Blockonomics
+ * @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ * International Registered Trademark & Property of Blockonomics
+ *}
+
+<div id="blockonomics-payment-helper"></div>
+<script>
+    let helper = document.getElementById("blockonomics-payment-helper")
```

Comment:
Adding script to handle the logo height looks like a good solution. We can rename the file to logo_height.tpl
We can also reduce the code as we are targeting a single img
```
var image = document.querySelector("img[src*='blockonomics/views/img']")
image.style.height = "{$blockonomicsLogoHeight}px"
```

Reply:
Updated. Working fine.